source/Halibut.Tests/Support/EnvironmentVariableReaderHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Castle.Core.Internal;
+
+namespace Halibut.Tests.Support
+{
+    public class EnvironmentVariableReaderHelper
+    {
+        public static bool EnvironmentVariableAsBool(string envVar, bool defaultValue)
+        {
+            var value = Environment.GetEnvironmentVariable(envVar);
+            if (string.IsNullOrEmpty(value))
+            {
+                return defaultValue;
+            }
+
+            return value!.Equals("true");
```

Comment:
Might be worth making this a case insensitive comparison. `bool.Parse` does this nicely 😁